src/tox.ini

```diff
@@ -32,6 +32,8 @@ deps =
 commands =
     pytest {posargs: \
         gravel/ }
+exclude =
+    gravel/cephadm
```

Comment:
oh, we can't be ignoring all of `gravel/cephadm/`. A lot of that is our own code. We should only ignore `gravel/cephadm/cephadm.bin`.

Reply:
Fixed